lib/iris/coord_systems.py

```diff
@@ -303,13 +291,37 @@ def as_cartopy_projection(self):
         )
 
     def as_cartopy_globe(self):
-        # Explicitly set `ellipse` to None as a workaround for
-        # Cartopy setting WGS84 as the default.
-        return ccrs.Globe(
-            semimajor_axis=self.semi_major_axis,
-            semiminor_axis=self.semi_minor_axis,
-            ellipse=None,
+        return self._crs.globe
+
+    def __getattr__(self, name):
+        if name == "semi_major_axis":
+            return self._crs.ellipsoid.semi_major_metre
+        if name == "semi_minor_axis":
+            return self._crs.ellipsoid.semi_minor_metre
+        if name == "inverse_flattening":
+            return self._crs.ellipsoid.inverse_flattening
+        if name == "datum":
+            datum = self._crs.datum.name
+            if datum.startswith("unknown") or datum.startswith("Unknown"):
```

Comment:
see other comment about "unknown" below. maybe this needs to be pulled into a documented subroutine

Reply:
[94ffbc5](https://github.com/SciTools/iris/pull/4704/commits/94ffbc5942ebc464539a846047799aae4dca83c8)